packages/app/src/locales/en.json

```diff
@@ -494,7 +512,12 @@
                     "required": "Zkvyper version is required"
                 },
                 "error": "Unable to get list of supported Zkvyper versions",
-                "details": "Details"
+                "details": "Details",
+                "evmVersionToTarget": {
+                    "label": "EVM Version to target",
+                    "custom": "Custom",
+                    "inputPlaceholder": "Custom version..."
+                }
```

Comment:
It seems that only a few of the new translations are being used. Let's remove the unused ones.

Please change the text from `EVM Version to target` to `EVM Version`, as it's clear what is meant here and the longer text overflows with other inputs on small resolutions.